crates/bevy_winit/src/winit_event.rs

```diff
@@ -61,6 +62,7 @@ pub enum WinitEvent {
     TouchInput(TouchInput),
 
     KeyboardInput(KeyboardInput),
+    KeyboardFocusLost(KeyboardFocusLost),
```

Comment:
The addition of this enum variant is a breaking change. Totally fine to do, but good to call out.

Reply:
I'm ready to provide all documentation necessary, but i'm not sure how exactly this is a breaking change, as in how it affects end users when they update? The new event is introduced, but it is only used internally and there is no need to change any existing code base. Or maybe a usage example is required?

Reply:
in other words just tell me what to do and i'll do it :)

Reply:
understood, thank you!